ts/features/bonus/cgn/__e2e__/cgn.e2e.ts

```diff
@@ -0,0 +1,85 @@
+import { ensureLoggedIn } from "../../../../__e2e__/utils";
+import I18n from "../../../../i18n";
+import { e2eWaitRenderTimeout } from "../../../../__e2e__/config";
+
+const CGN_TITLE = "Carta Giovani Nazionale";
+const CGN_BONUS_ITEM =
+  "Carta Giovani Nazionale (CGN) è l’incentivo per i giovani che favorisce la partecipazione ad attività culturali, sportive e ricreative, su tutto il territorio nazionale";
+
+const activateBonusSuccess = async () => {
+  const startActivationCta = element(by.id("activate-bonus-button"));
+  await waitFor(startActivationCta)
+    .toBeVisible()
+    .withTimeout(e2eWaitRenderTimeout);
+  await startActivationCta.tap();
+  await waitFor(element(by.text(I18n.t("bonus.cgn.activation.success.title"))))
+    .toBeVisible()
+    .withTimeout(e2eWaitRenderTimeout);
+};
+
+describe("CGN", () => {
+  beforeEach(async () => {
+    await device.reloadReactNative();
+    await ensureLoggedIn();
+  });
+
+  describe("When the user want to start activation from bonus list", () => {
+    it("Should complete activation", async () => {
+      await element(by.text(I18n.t("global.navigator.wallet"))).tap();
+      await element(
+        by.text(I18n.t("wallet.newPaymentMethod.add").toUpperCase())
+      ).tap();
+      await element(by.text(I18n.t("wallet.methods.bonus.name"))).tap();
+      const cgnBonusItem = element(by.text(CGN_BONUS_ITEM));
+      await waitFor(cgnBonusItem)
+        .toBeVisible()
+        .withTimeout(e2eWaitRenderTimeout);
+      await cgnBonusItem.tap();
+      await activateBonusSuccess();
+    });
+  });
+
+  describe("When the user want to start activation from card carousel", () => {
+    it("Should complete activation", async () => {
+      await element(by.text(I18n.t("global.navigator.wallet"))).tap();
+      await element(by.text(CGN_TITLE)).tap();
+      await activateBonusSuccess();
+    });
+  });
+
+  describe("When the user want to start activation from service detail", () => {
+    it("Should complete activation", async () => {
+      await element(by.text(I18n.t("global.navigator.services"))).tap();
+      const cgnServiceItem = element(by.text(CGN_TITLE));
+      await waitFor(cgnServiceItem)
+        .toBeVisible()
+        .withTimeout(e2eWaitRenderTimeout);
```

Comment:
This is failing locally. Could you double check please? Thanks!

Reply:
[5e7549b](https://github.com/pagopa/io-app/pull/3898/commits/5e7549b745239a80b0ed1bf329b1a097775c2a43) should have fixed

Reply:
Confirmed!